src/content/devices/multi/index.html

```diff
@@ -37,34 +37,39 @@
 
     <div id="sources">
 
-     <div>
-       <h2>getUserMedia():</h2>
-       <video class="gum" title="From getUserMedia()" autoplay controls></video>
-       <audio class="gum" title="From getUserMedia()" controls></audio>
-     </div>
-
-     <div>
-       <h2>Local media files:</h2>
-       <video title="From local video file" controls loop poster="images/poster.jpg" preload="metadata">
-         <source src="video/chrome.webm" type="video/webm" />
-         <source src="video/chrome.mp4" type="video/mp4" />
-         <p>This browser does not support the video element.</p>
-       </video>
-       <audio title="From local audio file" controls loop>
-         <source src="audio/audio.mp3" type="audio/mp3" />
-         This browser does not support the audio element.
-       </audio>
-     </div>
+      <div>
+        <h2>getUserMedia():</h2>
+        <div class='box'>
+          <video class="gum" title="audio and video stream from getUserMedia()" autoplay controls></video>
+          <label>Select audio output: </label>
+        </div>
+        <div class='box'>
```

Comment:
Rather than a box, I think you could just keep the select fairly close to the element it belongs with — say 10px vertical margin below the video or audio element, 20px margin below the select.